tox.ini

```diff
@@ -9,7 +9,13 @@ envlist=
 [flake8]
 max-line-length= 100
 exclude=
-ignore=E252
+ignore =
+    # E252 missing whitespace around parameter equals
+    # Already used too much in the codebase at the point of introduction.
+    E252,
+    # W504 line break after binary operator
+    # It's either this or W503 (line break _before_ binary operator), pick your poison
+    W504
```

Comment:
I think we'd rather ignore `W503` here if it's not a huge change to this PR:

That's actually flake8's preferred solution: https://www.flake8rules.com/rules/W503.html

See a nice summary here: https://github.com/raiden-network/raiden/pull/1382
(Roughly, ignoring W503 is flake8's default, but when we started ignoring E252, then flake8 stopped ignoring W503, so we need to add it back again)